elyra/metadata/schemas/code-snippet.json

```diff
@@ -23,11 +23,18 @@
         },
         "language": {
           "description": "Code snippet implementation language",
-          "type": "string"
+          "type": "string",
+          "ui_hints": {
+            "field_type": "dropdown",
```

Comment:
I thought these were going to have a `label` field as well?

Reply:
ah yes, I forgot that

Reply:
fixed